thirdparty/thorvg/inc/thorvg.h

```diff
@@ -1,6 +1,7 @@
 #ifndef _THORVG_H_
 #define _THORVG_H_
 
+#include <cstdint>
```

Comment:
CC @mgrudzinska 
This seems to be included already in thorvg `main` but not in `v0.15.x`, would be good if it was included in 0.15.10 or 0.16.0 if such releases are planned.
https://github.com/thorvg/thorvg/blob/v0.15.x/inc/thorvg.h

Reply:
ofc, I'll let you know!

Reply:
@akien-mga `#include <cstdint>` has been included in v0.15.10, Thanks.
https://github.com/thorvg/thorvg/releases/tag/v0.15.10

Reply:
Thanks!